packages/test/test-service-load/src/nodeStressTest.ts

```diff
@@ -97,10 +99,24 @@ async function orchestratorProcess(
         undefined,
         args.browserAuth);
 
-    // Create a new file if a testId wasn't provided
-    const url = args.testId !== undefined
-        ? await testDriver.createContainerUrl(args.testId)
-        : await initialize(testDriver, seed, profile, args.verbose === true);
+    // If testId is provided, then try to load the file first.
+    let url;
+    if (args.testId !== undefined) {
+        try {
+            url = await testDriver.createContainerUrl(args.testId);
+        } catch(e: any) {
+            if (!(e instanceof NonRetryableError)
+                    || (e.errorType !== DriverErrorType.fileNotFoundOrAccessDeniedError)) {
+                throw e;
+            }
+            // If file not found, then try to create.
+        }
```

Comment:
Instead of try-catch with specific error (which could change at some point), how about adding a function `testDriver.exists` to determine whether to create or not?

Reply:
@markfields Added doesDocumentExists in test driver.
Implemented only for odsp & local drivers, its not implemented for tinylicious & routerlicious drivers in this PR.